core/src/test/java/org/apache/accumulo/core/file/streams/RateLimitedInputStreamTest.java

```diff
@@ -56,22 +69,22 @@ public int read() {
     @Override
     public void seek(long pos) {
       throw new UnsupportedOperationException("Not supported yet."); // To change body of generated
```

Comment:
The comment seems to be from you IDE and not relevant to the code.  Please update or remove,